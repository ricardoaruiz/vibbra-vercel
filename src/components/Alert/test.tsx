import { renderWithThemeProvider } from 'utils/test'

import { Alert } from '.'

describe('<Alert />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(
      <Alert message="Altert message" />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
