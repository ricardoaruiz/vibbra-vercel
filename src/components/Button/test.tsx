import { renderWithThemeProvider } from 'utils/test'
import { Button } from '.'

describe('<Button />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Button />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
