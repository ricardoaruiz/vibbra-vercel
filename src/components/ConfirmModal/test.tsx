import { renderWithThemeProvider } from 'utils/test'

import { ConfirmModal } from '.'

describe('<ConfirmModal />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<ConfirmModal />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
