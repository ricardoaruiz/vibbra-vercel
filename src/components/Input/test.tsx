import { renderWithThemeProvider } from 'utils/test'

import { Input } from '.'

describe('<Input />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Input />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
