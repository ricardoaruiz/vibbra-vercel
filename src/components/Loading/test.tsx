import { renderWithThemeProvider } from 'utils/test'

import { Loading } from '.'

describe('<Loading />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Loading />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
