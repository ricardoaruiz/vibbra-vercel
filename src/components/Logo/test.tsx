import { renderWithThemeProvider } from 'utils/test'

import { Logo } from '.'

describe('<Logo />', () => {
  it('should render correctly', () => {
    const { container } = renderWithThemeProvider(<Logo />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
