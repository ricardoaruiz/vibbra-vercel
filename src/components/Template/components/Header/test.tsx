import { renderWithThemeProvider } from 'utils/test'

import { Header } from '.'

describe('<Header />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Header />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
