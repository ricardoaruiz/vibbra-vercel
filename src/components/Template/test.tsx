import { renderWithThemeProvider } from 'utils/test'

import { Template } from '.'

jest.mock('next/router', () => ({
  useRouter: () => ({
    push: jest.fn()
  })
}))

describe('<Template />', () => {
  it('should render the heading', () => {
    const { container } = renderWithThemeProvider(<Template />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
