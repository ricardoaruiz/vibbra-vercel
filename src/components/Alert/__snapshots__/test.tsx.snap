// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Alert /> should render the heading 1`] = `
.c0 {
  position: fixed;
  left: 0;
  width: 100%;
  color: #FAFAFA;
  padding: 1rem 2rem;
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
  background-color: #0cac21;
  bottom: -20px;
  opacity: 0;
  visibility: hidden;
}

<div
  class="c0"
>
  <h1>
    Altert message
  </h1>
</div>
`
